geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/HBaseDataStoreCommand.scala

```diff
@@ -21,9 +25,26 @@ trait HBaseDataStoreCommand extends DataStoreCommand[HBaseDataStore] {
   override def params: HBaseParams
 
   override def connection: Map[String, String] = {
+
+    val HBASE_SITE_FILE_NAME = "hbase-site.xml"
+    val HBASE_ZOOKEEPER_QUORUM = "hbase.zookeeper.quorum"
+
+    val zkParam: String = if (params.zookeepers != null) {
+      // Command line takes precedence
+      params.zookeepers
+    } else {
+      val conf = new Configuration()
```

Comment:
still need to look at how this interacts with the other resources...i'll comment soon.